subprojects/docs/src/docs/userguide/publishingIvy.adoc

```diff
@@ -209,14 +209,14 @@ The created task is named “`publish«_PUBNAME_»PublicationTo«_REPONAME_»Rep
 
 
 [[sec:the_publish_lifecycle_task]]
-==== The “`publish`” lifecycle task
+==== The `publish` lifecycle task
 
-The “`publish`” plugin (that the “`ivy-publish`” plugin implicitly applies) adds a lifecycle task that can be used to publish all publications to all applicable repositories named “`publish`”.
+The `publish` plugin (that the Ivy Publish Plugin implicitly applies) adds a lifecycle task that can be used to publish all publications to all applicable repositories named `publish`.
 
 In more concrete terms, executing this task will execute all api:org.gradle.api.publish.ivy.tasks.PublishToIvyRepository[] tasks in the project. This is usually the most convenient way to perform a publish.
 
 ++++
-<sample dir="ivy-publish/quickstart" id="publishingIvyPublishLifecycle" title="Publishing all publications via the “publish” lifecycle task">
+<sample dir="ivy-publish/quickstart" id="publishingIvyPublishLifecycle" title="Publishing all publications via the `publish` lifecycle task">
```

Comment:
The backticks will appear in the HTML. I just wanted to warn you. Leave the backticks there. Hopefully we'll have a solution some time down the line.